pipelines/controller/src/main/java/org/openmrs/analytics/metrics/StatsHelper.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openmrs.analytics.metrics;
+
+import org.apache.beam.sdk.metrics.GaugeResult;
+import org.apache.beam.sdk.metrics.MetricQueryResults;
+import org.apache.beam.sdk.metrics.MetricResult;
+import org.openmrs.analytics.MetricsConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class StatsHelper {
+  private static final Logger logger = LoggerFactory.getLogger(StatsHelper.class.getName());
+
+  public static Stats createStats(MetricQueryResults metricQueryResults) {
```

Comment:
nit: I don't mind if you move this static method to `Stats` class and remove `StatsHelper` to have less classes; but up to you. Please consider adding javadoc for this public method too, especially documenting how a `null` argument is handled. I would also go an additional step and annotate the argument as `@Nullable`.

Reply:
Refactored and added javadoc

Reply:
Great, thanks.